core/src/wallet/tezos/explorers/NodeTezosLikeBlockchainExplorer.cpp

```diff
@@ -289,5 +289,24 @@ namespace ledger {
                                                             _http,
                                                             getRPCNodeEndpoint());
         }
+
+        Future<bool> NodeTezosLikeBlockchainExplorer::isFunded(const std::string &address) {
+            return
+                _http->GET(fmt::format("blockchain/{}/{}/account/{}", address))
+                    .json().map<bool>(getExplorerContext(), [=](const HttpRequest::JsonResult &result) {
+                        auto& json = *std::get<1>(result);
+
+                        // look for the is_funded field
+                        //Is there a fees field ?
+                        auto field = "is_funded";
+                        if (!json.IsObject() || !json.HasMember(field) ||
+                            !json[field].IsBool()) {
+                            throw make_exception(api::ErrorCode::HTTP_ERROR,
+                                                 "Failed to get is_funded from network, no (or malformed) field \"result\" in response");
+                        }
+
+                        return json[field].GetBool();
```

Comment:
Since that chunk of code seems to be the exact same than the one in `ExternalTezosLikeBlockchainExplorer`, can you put the code in a free function instead ?

Reply:
No, because even though it’s similar, it might change regarding what we do on an explorer or the other – and I’m not completely sure `is_funded` is present on our explorers. I wait for Chenyu for this.